week-1/Homework/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,14 @@
-// Write your function here
 
-var greeting = createGreeting("Daniel");
+
+function createGreeting (name) {
+    const message = "Hello, my name is " + name;
+    return message;
+}
+
+var greeting = createGreeting("Burçak");
 
 console.log(greeting);
+
+
+
+/* Hello, my name is Burçak */
```

Comment:
Minor: for the sake of simplicity, you can use `//` instead of `/*` + `*/` to comment one single line. `/*` + `*/` is normally used to comment big chunks of code.